src/plugins/peer-view/public/js/peerview.js

```diff
@@ -252,14 +265,21 @@
           case 'mission-control-register':
             p.userName = msg[1];
             break;
+          case 'sys.ping':
+            p.sendemit('sys.pong', msg[1]);
+            break;
```

Comment:
Here we simulate the response from the ROV  to a ping request. This will cause the client to calculate the latency as that to the host instead of the ROV